gallery/how-to/working-with-local-psrepositories.md

```diff
@@ -0,0 +1,165 @@
+
+> [!NOTE]
+> You must create an account in the NuGet server, then sign in to generate and save the API key.
+> For a file share, use any non-blank string for the NuGetApiKey value.
+
+Example commands are:
+
+```powershell
+# Publish to a NuGet Server repository using my NuGetAPI key
+Publish-Module -Path 'c:\projects\MyModule' -Repository LocalPsRepo -NuGetApiKey 'oy2bi4avlkjolp6bme6azdyssn6ps3iu7ib2qpiudrtbji'
+
+# Publish to a file share repo - the NuGet API key must be a non-blank string
+Publish-Module -Path 'c:\projects\MyModule' -Repository LocalPsRepo -NuGetApiKey 'AnyStringWillDo'
+```
+
+> [!IMPORTANT]
+> To ensure security, API keys should not be hard-coded in scripts. Use a secure key management
+> system.
+
+## Installing PowerShellGet on a disconnected system
+
+Deploying PowerShellGet is difficult in environments that require systems to be disconnected from
+the internet. PowerShellGet has a bootstrap process that installs the latest version the first
+time it's used. The OfflinePowerShellGetDeploy module in the PowerShell Gallery provides cmdlets
+that support this bootstrap process for your local repository.
+
+To bootstrap an offline deployment, you need to:
+
+- Download and install the OfflinePowerShellGetDeploy your internet-connected system and your
+  disconnected systems
+- Download PowerShellGet and its dependencies on the internet-connected system using the
+  `Save-PowerShellGetForOffline` cmdlet
+- Copy PowerShellGet and its dependencies from the internet-connected system to the disconnected
+  system
+- Use the `Install-PowerShellGetOffline` on the disconnected system to place PowerShellGet and its
+  dependencies into the proper folders
+
+The following commands use `Save-PowerShellGetForOffline` to put all the components into a folder
+`f:\OfflinePowerShellGet`
+
+```powershell
+# Download the OfflinePowerShellGetDeploy to a location that can be accessed
+# by both the connected and disconnected systems.
+Save-Module -Name OfflinePowerShellGetDeploy -Path 'F:\' -Repository PSGallery
+Import-Module F:\OfflinePowerShellGetDeploy
+
+# Put PowerShellGet somewhere locally
+Save-PowerShellGetForOffline -LocalFolder 'F:\OfflinePowerShellGet'
+```
+
+At this point, you must make the contents of `F:\OfflinePowerShellGet` available to your
+disconnected systems. Run the `Install-PowerShellGetOffline` cmdlet to install PowerShellGet on the
+disconnected system.
+
+> [!NOTE]
+> It is important that you do not run PowerShellGet in the PowerShell session before running these
+> commands. Once PowerShellGet is loaded into the session, the components cannot be updated. If you
+> do start PowerShellGet by mistake, exit and restart PowerShell.
+
+```powershell
+Import-Module F:\OfflinePowerShellGetDeploy
+
+Install-PowerShellGetOffline -LocalFolder 'F:\OfflinePowerShellGet'
+```
+
+After running these commands, you are ready to run `Register-PSRepository` and use
+PowerShellGet with your local repository.
```

Comment:
After checking in, it seems that the idea here is that you can then set Offline PowerShellGet as an additional PSRepository, Maybe would be more clear if we said:
After running these commands, you are ready to register your local folder "F:\OfflinePowerShellGet" as a PSRepository by running 'Register-PSRepository', as described in the section "Creating a Local Repository" above (maybe we can link to that section here).